Mathlib/Data/Int/Cast/Lemmas.lean

```diff
@@ -234,20 +234,20 @@ theorem ext_int [AddMonoid A] {f g : ℤ →+ A} (h1 : f 1 = g 1) : f = g :=
 
 variable [AddGroupWithOne A]
 
-theorem eq_int_cast_hom (f : ℤ →+ A) (h1 : f 1 = 1) : f = Int.castAddHom A :=
+theorem eq_intCast_hom (f : ℤ →+ A) (h1 : f 1 = 1) : f = Int.castAddHom A :=
```

Comment:
You should only change the lemma statements that actually mention `intCast` in the statement. This one is about `Int.castAddHom`, so it should maybe be renamed `eq_castAddHom` or something

Reply:
Ok, but does this mean that `map_int_cast` below stays `map_int_cast` and doesn't become `map_intCast` after all?

Reply:
No, I think that one is right. Chris is pointing out that the current lemma is not about `intCast`.
I do think that this name should mention int somehow. Because there's presumably a similar lemma for nat?

Reply:
Changed it to `eq_int_castAddHom`. I didn't find the similar lemma for nat.

However for the other `eq_intCast` and `map_intCast` there are the similar lemmas for nat and rat.